docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md

```diff
@@ -13,51 +13,36 @@ For an introduction to templating and template variables, refer to the [Templati
+| `Statistics`              | Returns a list of all the standard statistics.                                                                                                                                |
 
 For details about the metrics CloudWatch provides, please refer to the [CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/CW_Support_For_AWS.html).
 
-## Example of templated queries
-
-Here is an example of the dimension queries which will return list of resources for individual AWS Services:
+### Using variables in queries
 
-| Query                                                                                                                         | Service          |
-| ----------------------------------------------------------------------------------------------------------------------------- | ---------------- |
-| `dimension_values(us-east-1,AWS/ELB,RequestCount,LoadBalancerName)`                                                           | ELB              |
-| `dimension_values(us-east-1,AWS/ElastiCache,CPUUtilization,CacheClusterId)`                                                   | ElastiCache      |
-| `dimension_values(us-east-1,AWS/Redshift,CPUUtilization,ClusterIdentifier)`                                                   | RedShift         |
-| `dimension_values(us-east-1,AWS/RDS,CPUUtilization,DBInstanceIdentifier)`                                                     | RDS              |
-| `dimension_values(us-east-1,AWS/S3,BucketSizeBytes,BucketName)`                                                               | S3               |
-| `dimension_values(us-east-1,CWAgent,disk_used_percent,device,{"InstanceId":"$instance_id"})`                                  | CloudWatch Agent |
-| `resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})` | ELB              |
-| `resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"Component":["$service"],"Environment":["$environment"]})`        | ELB              |
-| `resource_arns(eu-west-1,ec2:instance,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})`                      | EC2              |
+Refer to the [variable syntax documentation]({{< relref "../../variables/syntax.md" >}}).
 
 ## Using JSON format template variables
 
 Some queries accept filters in JSON format and Grafana supports the conversion of template variables to JSON.
 
-If `env = 'production', 'staging'`, following query will return ARNs of EC2 instances which `Environment` tag is `production` or `staging`.
+If `env = 'production', 'staging'`, a `Resource ARNs` query with the following filter will return ARNs of EC2 instances where the `Environment` tag is equal to `production` or `staging`.
```

Comment:
I am missing a "For example, if ...` at the beginning here.

Reply:
done